sklearn/feature_selection/rfe.py

```diff
@@ -80,6 +80,10 @@ class RFE(BaseEstimator, MetaEstimatorMixin, SelectorMixin):
         ranking position of the i-th feature. Selected (i.e., estimated
         best) features are assigned rank 1.
 
+    scores_ : list
+        List of step scores, it is used when implementing ``RFECV``.
```

Comment:
That doesn't seem correct, right? It is set when calling RFE.fit() below, right?

Reply:
Thanks for the comment @amueller 
`RFE.fit()` always returns: `self._fit(X, y)` [see here](https://github.com/nishnik/scikit-learn/blob/a6b143358b48251cb68dadbcb25d97d1fee0bc7a/sklearn/feature_selection/rfe.py#L143)
And [here](https://github.com/nishnik/scikit-learn/blob/a6b143358b48251cb68dadbcb25d97d1fee0bc7a/sklearn/feature_selection/rfe.py#L149) it's already written that 
> self.scores_ will not be calculated when calling _fit through fit